config/jail.conf

```diff
@@ -761,6 +761,7 @@ logpath  = /var/log/fail2ban.log
 banaction = %(banaction_allports)s
 bantime  = 1w
 findtime = 1d
+protocol = all
```

Comment:
This will enable also connection-less protocol like UDP, I don't think it would be good idea per default,
see waning in https://github.com/fail2ban/fail2ban/blob/0.10/config/jail.conf#L668